repair/repair.cc

```diff
@@ -319,10 +319,13 @@ tracker::tracker(size_t max_repair_memory)
 }
 
 void tracker::start(repair_uniq_id id) {
+    _pending_repairs.insert(id.uuid);
```

Comment:
I seems to me you added more sets than really needed. Instead of adding additional sets, like this _pending_repairs, wouldn't it make sense to just have additional states in the already-existing _status map? E.g., _status = PENDING (or STARTING) can perhaps mean the same thing that your new _pending_repairs was trying to say?

Reply:
std::unordered_map<int, repair_status> _status

is something we expose to the user API which has 3 states: RUNNING, SUCCESSFUL, FAILED.

I do not want to mess up / break the api that's why I introduced new fields to track the abort and only for the abort.